Featured

Jace0615 opened the pull request:
main page banner test added

Jace0615 committed:
featured
fix indent

Jace0615 committed:
revert indent

Jace0615 committed:
featured / some code adjust
1. search.aspec.js -> agit, collection, nft location rewrights
2. nft.spec.js -> nft location rewrights

Jace0615 committed:
featured / some directories change

Jace0615 committed:
featured/
add new tests
1. proposal commen time stamp check
2. proposal voting result check

Jace0615 committed:
makefile code fix

Jace0615 committed:
featured/
main page banner test added

hackartists commented on tests/agit.spec.js:
```diff
@@ -235,7 +235,7 @@ test.describe.serial("Agit", () => {
     });
     await page.press('textarea[placeholder="댓글을 남겨주세요."]', "Enter");
-    await page.waitForTimeout(latency);
+    await page.waitForTimeout(3000);
```
이거는 이렇게 길게 잡은 이유가 뭔가요? 이제 트랜잭션과 fetcher 문제면 차라리 longLatency 하나 잡고 파라미터로 줄 수 있게 구성하는게 좋을거 같긴하네요.

Jace0615 committed:
featured/ added tests
1. check the blur when in to the create page
2. check the size of NFT image sections when click 'Detail', 'Activity' menu
